### PR TITLE
fix error package name

### PR DESCRIPTION
**Pull request by yiyang5055**
/kind cleanup
/sig-scheduling

**What type of PR is this?**

/kind cleanup

**What this PR does / why we need it**:

fix error package name

**Which issue(s) this PR fixes**:

Fixes #

**Special notes for your reviewer**:

**Does this PR introduce a user-facing change?**:

```release-note
None
```

**Additional documentation e.g., KEPs (Kubernetes Enhancement Proposals), usage docs, etc.**:

```docs
None
```

### CODE REVIEW

**Comment by yiyang5055** [comment]
pinging @ahg-g

File: pkg/scheduler/framework/plugins/noop/noop.go [hunk 1]
```diff
@@ -23,27 +23,28 @@ import (
 	framework "k8s.io/kubernetes/pkg/scheduler/framework/v1alpha1"
 )
 
-// NoOpFilter is a plugin that implements the filter plugin and always returns Success.
+// NoFilter is a plugin that implements the filter plugin and always returns Success.
 // This is just to make sure that all code dependencies are properly addressed while
 // working on legitimate plugins.
-type NoOpFilter struct{}
+// Note: The struct cannot be named NoOpFilter, otherwise the golint check cannot pass
+type NoFilter struct{}
```
Comment (alculquicondor) [inline pkg/scheduler/framework/plugins/noop/noop.go]: just name it `Filter`

**Comment by alculquicondor** [comment]
Feel free to amend/squash in this case, given that the PR is small enough.

**Review by draveness — COMMENTED** [review]
/hold feel free to cancel

Please squash the commits into one

File: pkg/scheduler/framework/plugins/noop/noop.go (hunk 1 above)
Comment (ahg-g) [reply pkg/scheduler/framework/plugins/noop/noop.go]: +1 to naming it Filter since the package name should give enough context.

Comment (yiyang5055) [reply pkg/scheduler/framework/plugins/noop/noop.go]: > +1 to naming it Filter since the package name should give enough context.

OK

**Comment by yiyang5055** [comment]
> /hold feel free to cancel
> 
> Please squash the commits into one

ok, i understand

**Pull request merged**